pom.xml

```diff
@@ -12,13 +12,13 @@
     <properties>
         <java.version>17</java.version>
         <vaadin.version>24.2.0.alpha7</vaadin.version>
-        <selenium.version>4.10.0</selenium.version>
+        <selenium.version>4.11.0</selenium.version>
```

Comment:
remove this line since it's not need when they fixed the problem in this commit https://github.com/spring-projects/spring-boot/commit/d4031f3424331255b3ca72e11adb2c62252f4205

Reply:
What is fixed by Spring Boot changing to use the BOM? Does TestBench override it?

Reply:
Yes, `selenium-bom` included in now in `spring-boot-starter-parent` instead of defining versions, is overriden by `vaadin-bom` included in starter project